public/doc/swagger-2-v0.1.0.yaml

```diff
@@ -2534,6 +2537,9 @@ definitions:
         format: date-time
         readOnly: true
         type: string
+      mac_addresses:
+        readOnly: true
+        type: string
```

Comment:
@Ladas should this be an array of strings?

Reply:
hm, yes it should

Reply:
hum I need to change the generator